docker/compose.yml

```diff
@@ -141,11 +141,12 @@ services:
     restart: "no"
     environment:
       LOG4J_CONFIGURATION_FILE: /var/lib/besu/log4j.xml
+      JAVA_OPTS: -Xmx1536m -XX:+UnlockExperimentalVMOptions -XX:-UseG1GC -XX:+UseZGC
```

Comment:
@Filter94 reduced this from 2GB to 1.5GB, tested it locally and should be safe for max heap size of l2-node-besu

Reply:
Thanks! Let's see